### ISSUE TITLE
Extra zeros in token value on send confirmation screen

### ISSUE THREAD

**Issue by dpazdan** [MetaMask/metamask-extension#6762]
**Describe the bug**
User reported seeing 25000000 USD on the send screen, with the equivalent USD value, when sending 25 USDC.  I tested and confirmed this with a 0.9 USDC send (see screenshot below).  If the transaction is confirmed, it will still send the correct amount.

**To Reproduce**
1. Send USDC of any amount.
2. See inaccurate amount displayed (by 6 decimal places)

**Expected behavior**
See the correct amount prior to confirming.

**Screenshots**
![image](https://user-images.githubusercontent.com/16330048/60297261-6aeb7e00-98ed-11e9-8c69-eb1d3b571469.png)

**Additional context**
Support case #: 20866

**Comment by bdresser** [comment]
thanks @dpazdan, please holler if this comes up again

**Comment by Lastrellik** [comment]
This is happening for us too.

**Comment by realisation** [comment]
I've been encountering this with other ERC20's today as well. The amount of 0's are the amount of decimals on the ERC20.

**Comment by bdresser** [comment]
thanks @Lastrellik @realisation - what MM version are you on and what tokens are you seeing it with? cc @tmashuang

**Comment by Lastrellik** [comment]
@bdresser Version 6.7.0 and we have some test tokens deployed to Kovan that are showing this behavior. The addresses of those tokens can be seen here: 
```
User Facing Stable Coins
    Dai: 0x4C561F234520Bb75184E5337041285F990AA4107
    USDCoin: 0x49D9cebaDcD42F8a556e0a5a982a95dc191BF8c6
    TrueUSD: 0x936Bae0783D0e3c6B3799091E8803ffb563302e0
    Paxos Standard: 0xF63BF08CA72b7b4d0219777769f7B2E85C172BCB
    Gemini dollar: 0x6459d0c05A62dC1b4cC3b8f0CF652B6A2A0c3537
```
Give me an address and I can send you some of these tokens if it's useful for you. Let me know if you need more help reproducing.

**Comment by bdresser** [comment]
We've reproduced, digging in now

**Comment by danfinlay** [comment]
Fixed in Version 6.7.1, publishing now.
